.github/workflows/notifier.yml

```diff
@@ -0,0 +1,15 @@
+name: Notify users based on issue labels
+
+on:
+  issues:
+      types: [labeled]
+
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+    steps:
+        - uses: jenschelkopf/issue-label-notification-action@1.3
+          with:
+             recipients: |
+                  help wanted=@akihironitta
+                  documentation=@akihironitta
```

Comment:
I had in mind copying what's done in https://github.com/pytorch/pytorch/issues/24422

Otherwise, it is too much of a hassle to subscribe/unsubscribe (needs 3 reviews + CI).

If the action you chose does not support it, we should try and implement it.